### ISSUE TITLE
From node v17 plugin crashes 

### ISSUE THREAD

**Issue by djsomi** [SebastianOsinski/HomebridgePluginSamsungAirConditioner#38]
Hi All,

I just upgraded node from v16.18.0, and the plugin constantly crashing with this:

```
[02/11/2022, 08:26:51] Error: unsupported
    at configSecureContext (node:internal/tls/secure-context:279:15)
    at Object.createSecureContext (node:_tls_common:117:3)
    at Object.connect (node:_tls_wrap:1629:48)
    at AirConditionerApi.connect (/usr/lib/node_modules/homebridge-plugin-samsung-air-conditioner/lib/air-conditioner-api.js:49:27)
    at AirConditioner.getServices (/usr/lib/node_modules/homebridge-plugin-samsung-air-conditioner/lib/air-conditioner.js:38:18)
    at BridgeService.createHAPAccessory (/usr/lib/node_modules/homebridge/src/bridgeService.ts:461:41)
    at /usr/lib/node_modules/homebridge/src/server.ts:381:44
    at Array.forEach (<anonymous>)
    at Server.loadAccessories (/usr/lib/node_modules/homebridge/src/server.ts:298:29)
    at Server.start (/usr/lib/node_modules/homebridge/src/server.ts:179:12)
```
Any node version is higher  than v16.18.0 is not working. Any idea, how to solve this? 
For temporary just downgraded back to v16.18.0. to keep things working.

Thanks
Zoltan

**Comment by KJTamas** [comment]
Hi,

My friend have exactly same error. 

thx,
Tamas

**Comment by RamjetX** [comment]
Same thing,

I've attempted to change the SSL Cipher and Min protocols to Level 1 and v1.0 by editing: sudo nano /etc/ssl/openssl.cnf

No luck so far. Although granted I'm stabbing in the dark with thing kind of thing at the moment. But learning lots

**Comment by javipe74** [comment]
I'm experienced same error with node 18.x I had to go back to the latest version of node.js 16.x  that will go in EoS next September. Anyone has managed to solve the issue?.

**Comment by SebastianOsinski** [comment]
Hi everyone. I just released 3.0.0 version of the plugin. It should resolve all SSL issues on new Node versions. I tested it with current homebridge docker image and it works great.

**Comment by javipe74** [comment]
Thanks a lot @SebastianOsinski I have upgraded the plugin and updated to last 18.x node.js version and I confirm it works perfectly. You made my day
